### ISSUE TITLE
Data not displayed when using a wrapper component

### ISSUE THREAD

**Issue by YassienW** [FormidableLabs/victory#2128]
# Bug Reports

### Checklist

- [x] I am using the latest version of Victory

- [x] I have read through the [FAQ](https://formidable.com/open-source/victory/docs/faq) and [Guides](https://formidable.com/open-source/victory/guides/)
- [x] I've searched open issues to make sure I'm not opening a duplicate issue

### The Problem

When creating a component wrapper and rendering  VictoryLine inside, data isn't shown (even with all props passed)

### Reproduction

https://codesandbox.io/s/issue-1498-solution-forked-ps3w78?file=/index.js

Replacing LineWrapper with VictoryLine renders the data immediately

**Comment by becca-bailey** [comment]
Hi @YassienW, this sounds like an issue with the static role property. This is definitely one of the more confusing parts of Victory's internals, and I hope it won't always be necessary! Every Victory component (like `VictoryLine` has a static role property that determines which props it receives from the parent. Wrapper components also need this role in order to receive the right props.

I wasn't able to see your `LineWrapper` in the sandbox, but try setting `LineWrapper.role = 'line'` before passing it into your chart.

**Comment by YassienW** [comment]
ah i apologize, i forgot to save the sandbox. I have updated it now.

I noticed two things: 

![image](https://user-images.githubusercontent.com/11986226/155233578-593577ae-d22a-4d39-bbe8-8301d1ed5488.png)

1. The data is rendered, but it seems to start from the border of the chart area
2. The x-axis and y-axis labels are not using the values provided by the x and y props

I tried adding a role prop to the wrapper but nothing changed
